Stats/Supporting Files/zh-Hant.lproj/Localizable.strings

```diff
@@ -330,7 +330,7 @@
 "Every 12 hours" = "每 12 小時";
 "Every 24 hours" = "每 24 小時";
 "Network activity" = "網路活動";
-"Last reset" = "Last reset %0 ago";
+"Last reset" = "上次重置於 %0 前";
```

Comment:
Would `重設` be better?

Reply:
Thanks for your reply.

I know that the term "重設" is better than "重置" in Traditional Chinese, especially in Taiwan. However the word "Reset" is translated into "重置" by Apple in its OSs, in order to match Traditional Chinese translation with OS's translation and avoid confusing, that the reason why I choose "重置" for "Reset" instead of "重設".

Reply:
Okay, cool.